pkgs/tools/admin/awscli2/default.nix

```diff
@@ -20,8 +20,10 @@ let
         };
       });
       urllib3 = prev.urllib3.overridePythonAttrs (prev: {
-        format = "setuptools";
-        pyproject = null;
+        pyproject = true;
+        nativeBuildInputs = with python3.pkgs; [
+          setuptools
+        ];
         src = prev.src.override {
           version = "1.26.18";
```

Comment:
```suggestion
          inherit version;
```

Reply:
@andrevmatos  while its more correct, this was already like this on master, since awscli is an application, can we reduce number of changes to this PR just to get this merged sooner